client/sites/hooks/use-transfer-with-software-status-query.ts

```diff
@@ -37,12 +37,12 @@ export const useTransferWithSoftwareStatus = (
 		queryKey: [ 'software-transfer-status', siteId, atomicTransferId ],
 		queryFn: () => getTransferWithSoftwareStatus( siteId, atomicTransferId ),
```

Comment:
nit: should this variable be changed to transferId?

Reply:
Yes, I fixed it! Thanks